### PR TITLE
Lint CONTRIBUTORS.txt alphabetical order

### PR DESCRIPTION
**Pull request by pollydrag**
## What do these changes do?

Lint CONTRIBUTORS.txt alphabetical order. 
In CI and "make flake"

## Are there changes in behavior for the user?

Only lint, does not affect behavior.

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names. 
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -180,6 +181,12 @@ jobs:
     script:
     - python setup.py check --metadata --restructuredtext --strict --verbose
 
+  - <<: *_lint_base
+    env:
+    - TARGET="'CONTRIBUTORS.txt sorting check'"
+    script:
+    - LC_ALL=C sort -c CONTRIBUTORS.txt
```
[the remark this answers is not in the document]
Comment (pollydrag) [reply .travis.yml:189]: Yes, it print error message and return 1

File: .travis.yml [hunk 2]
```diff
@@ -180,6 +181,12 @@ jobs:
     script:
     - python setup.py check --metadata --restructuredtext --strict --verbose
 
+  - <<: *_lint_base
+    env:
+    - TARGET="'CONTRIBUTORS.txt sorting check'"
```
[the remark this answers is not in the document]
Comment (pollydrag) [reply .travis.yml]: :+1:  It look great:

image

File: .travis.yml [hunk 3]
```diff
@@ -37,9 +37,6 @@ _helpers:
   stage: &doc_stage_name docs, linting and pre-test checks
   <<: *_mainstream_python_base
   <<: *_reset_steps
-  install:
```
Comment (webknjaz) [inline .travis.yml]: Hm.. it's odd. can you rebase on top of master?

File: .travis.yml [hunk 4]
```diff
@@ -145,14 +142,12 @@ jobs:
     sudo: required
 
   - <<: *_doc_base
-    env:
-    - TARGET=docs
+    name: docs
```
Comment (webknjaz) [inline .travis.yml]: These changes are not related. Only edit what's related to your PR.

File: .travis.yml (hunk 3 above)
Comment (webknjaz) [reply .travis.yml]: You shouldn't delete this.

Comment (webknjaz) [reply .travis.yml]: Any reason you wanted to delete it?

File: .travis.yml [hunk 5]
```diff
@@ -162,24 +157,27 @@ jobs:
...
-    - TARGET="'dist setup check'"
+    name: dist setup check
     install:
     - *upgrade_python_toolset
     - pip install -r requirements/cython.txt
     - pip install -r requirements/ci.txt -r requirements/doc.txt
     script:
     - python setup.py check --metadata --restructuredtext --strict --verbose
 
+  - <<: *_lint_base
```
Comment (webknjaz) [inline .travis.yml:183]: you probably want `install: skip` here.

File: .travis.yml [hunk 6]
```diff
@@ -39,6 +39,7 @@ _helpers:
   <<: *_reset_steps
   install:
   - *upgrade_python_toolset
+  - pip install -U -r requirements/cython.txt
```
[the remark this answers is not in the document]
Comment (webknjaz) [reply .travis.yml]: Just override install with `install: skip` in job you're adding. I don't want to influence that part now. I'll take care of it later.

File: .travis.yml (hunk 5 above)
Comment (pollydrag) [reply .travis.yml:183]: :+1:

**Comment by asvetlov** [comment]
@webknjaz please merge when ready

**Comment by webknjaz** [comment]
Sure

**Comment by webknjaz** [comment]
@pollydrag thanks!
